PatchesApi.Tests/V1/Boundary/Validation/UpdatePatchQueryValidatorTests.cs

```diff
@@ -0,0 +1,53 @@
+using FluentValidation.TestHelper;
+using PatchesApi.V1.Boundary.Request;
+using PatchesApi.V1.Boundary.Request.Validation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PatchesApi.Tests.V1.Boundary.Validation
+{
+    public class UpdatePatchQueryValidatorTests
+    {
+        private readonly UpdatePatchQueryValidator _classUnderTest;
+
+        public UpdatePatchQueryValidatorTests()
+        {
+            _classUnderTest = new UpdatePatchQueryValidator();
+        }
+
+        [Fact]
+        public void RequestShouldErrorWithNullId()
+        {
+            var query = new UpdatePatchesResponsibilityRequest();
```

Comment:
As before. Explicitly set Id to null

Reply:
I don't think we can do that as Id is a Guid value so I can't set it to null